crates/driver/src/boundary/mod.rs

```diff
@@ -48,12 +48,21 @@ fn web3(eth: &Ethereum) -> Web3 {
     eth.contracts().settlement().raw_instance().web3()
 }
 
-/// Builds a web3 client that bufferes requests and sends them in a
+/// Builds a web3 client that buffers requests and sends them in a
 /// batch call.
 pub fn buffered_web3_client(ethrpc: &Url) -> Web3 {
+    web3_client(ethrpc, 20, 10)
+}
+
+/// Builds a web3 client that sends requests one by one.
+pub fn unbuffered_web3_client(ethrpc: &Url) -> Web3 {
+    web3_client(ethrpc, 0, 0)
```

Comment:
I think this should rather use the regular http transport directly than the buffered transport with settings that essentially disables buffering.